boilerplates/react-union-boilerplate-basic/src/apps/SampleApp/components/Root/Root.js

```diff
@@ -5,6 +5,6 @@ import routes from '../../routes';
 
 import './Root.scss';
 
-const Root = () => <Union routes={routes} />;
+const Root = props => <Union routes={routes} {...props} />;
```

Comment:
I could not figure out why my code was not working until I found this culprit. We should add some kind of check or warning that detects THIS.